src/main/java/com/ibm/janusgraph/utils/importer/vertex/VertexLoaderWorker.java

```diff
@@ -90,11 +90,19 @@ private void acceptRecord(Map<String, String> record) throws Exception {
             }
 
             // Update property only if it does not exist already
-            if (!v.properties(propName).hasNext()) {
-                // TODO Convert properties between data types. e.g. Date
-                Object convertedValue = BatchHelper.convertPropertyValue(value,
-                        graphTransaction.getPropertyKey(propName).dataType());
-                v.property(propName, convertedValue);
+            // Removes the vertex in catch block if uniqueness constraint fails for a key and value 
+            try {
```

Comment:
Perhaps expand the try catch scope by moving the "try {" to line#74 and the catch block outside the for loop to catch all property related errors.

Reply:
agree with @tedhtchang . cover the for loop with try/catch would be better